src/Stoa.ts

```diff
@@ -828,6 +850,23 @@ class Stoa extends WebService
                     reject(err);
                 }
             }
+            else if (stored_data.type === "transaction")
+            {
+                try
+                {
+                    let tx = Transaction.reviver("", stored_data.data);
+                    let changes = await this.ledger_storage.putTransactionPool(tx);
```

Comment:
`changes ` is not used.

Reply:
Using the changes, I modified the code in many areas.
Validates the transaction inserted and the number of DB reflections of outputs & inputs